test/adapter.spec.js

```diff
@@ -96,6 +105,8 @@ describe('adapter', () => {
     expect(adapter.getObserver(foo, 'bar', Object.getPropertyDescriptor(foo, 'bar'))).not.toBe(null);
     expect(adapter.getObserver(foo, 'baz', Object.getPropertyDescriptor(foo, 'baz'))).toBe(null);
     expect(adapter.getObserver(foo, 'xup', Object.getPropertyDescriptor(foo, 'xup'))).not.toBe(null);
+    expect(adapter.getObserver(foo, 'strict1', Object.getPropertyDescriptor(foo, 'strict1'))).not.toBe(null);
+    expect(adapter.getObserver(foo, 'strict2', Object.getPropertyDescriptor(foo, 'strict2'))).not.toBe(null);
```

Comment:
@cnaj, what is the purpose of this strict2 expectation? It seems nearly identical to previous line that uses strict1 instead of strict2.